### ISSUE TITLE
Insufficient color contrast of tables on "how to use" page

### ISSUE THREAD

**Issue by untill** [it-vegard/a11y-reference-website#96]
TSIA
https://www.clothes4all.net/en/how-to

Reported by Andreas G.
